dbus/methods.go

```diff
@@ -291,6 +291,8 @@ func (c *Conn) listUnitsInternal(f storeFunc) ([]UnitStatus, error) {
 // ListUnits returns an array with all currently loaded units. Note that
 // units may be known by multiple names at the same time, and hence there might
 // be more unit names loaded than actual units behind them.
+// Also note that a unit is only loaded if it is active and/or enabled.
+// Disabled and inactive units will thus not be returned.
```

Comment:
My understanding it that this last line should be "Units that are both disabled and inactive...", right? That is, this is a strong logical AND.